Sprout/Models/Post.h

```diff
@@ -17,6 +17,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) NSNumber *completedTasks;
 @property (nonatomic) NSNumber *totalTasks;
 @property (nonatomic) NSString *caption;
+@property (nonatomic) NSArray *usersWhoLiked;
```

Comment:
nit: Assuming array of users but lightweight generics would be good here since maybe it's user identifiers or something.